src/components/ApplicantForm/ApplicantForm.test.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { render, screen, fireEvent, findByText, waitFor} from "@testing-library/react";
+import ApplicantForm from "./ApplicantForm";
+import { mocked } from 'ts-jest/utils';
+import '@testing-library/jest-dom'
+import { MemoryRouter } from "react-router-dom";
+import { AuthContext } from '../../contexts/index'
+import { getAttributes } from '../../assets/api-calls'
+
+jest.mock('../../assets/api-calls')
+
+describe('ApplicantForm', () => {
+  it('should render a form', async () => {
```

Comment:
I believe if this test is just that a form renders it would be sufficient to declare the inputs and expect that they are in the document.

I think another test could be `'Should allow a user to input information'` or something and then that test could use the fireEvent.change to enter information into the form. Then you could use findByText to get the entered information.